src/styles/_page-layout.scss

```diff
@@ -1,4 +1,4 @@
-.curiosity, .curiosity-page-messages, .curiosity-page-toolbar {
+.curiosity, .curiosity-page-messages, .curiosity-page-toolbar, .curiosity-tabs-container {
```

Comment:
required:
This needs to align towards `.curiosity-page-section` for consistency